.gitignore

```diff
@@ -22,3 +22,4 @@ yarn-error.log*
 package-lock.json
 
 .idea/*
+.obsidian
```

Comment:
was this meant to be added to this PR?

Reply:
Nope! Good catch. It won't cause issues, but it also doesn't need to be there 🙂

Reply:
Yes! It's an editor I sometimes use. No need to sync the settings to the repo 🙂